docs/annexes/annex-06-pid-rulebook.md

```diff
@@ -130,24 +132,50 @@ number in the domestic namespace.
 EXAMPLE: The first domestic PID namespace for Germany could be
 "eu.europa.ec.eudi.pid.de.1".
```

Comment:
```suggestion
For instance, the first domestic PID namespace for Germany would be
"eu.europa.ec.eudi.pid.de.1".
```

Reply:
I'd rather not change the existing text in this PR.